backend/internal/service/webauthn_service.go

```diff
@@ -224,17 +232,25 @@ func (s *WebAuthnService) VerifyLogin(ctx context.Context, sessionID string, cre
 		tx.Rollback()
 	}()
 
-	// Load & delete the session row
+	// Load the session row first (MySQL doesn't support RETURNING in DELETE)
 	var storedSession model.WebauthnSession
 	err := tx.
 		WithContext(ctx).
-		Clauses(clause.Returning{}).
-		Delete(&storedSession, "id = ?", sessionID).
+		First(&storedSession, "id = ?", sessionID).
```

Comment:
Do we need to explicitly READ LOCK the row?

Reply:
Ditto as above